content/ytt/docs/latest/ytt-schema.md

```diff
@@ -0,0 +1,60 @@
+---
+title: ytt @schema/match
+---
+
+## Overview
+
+`ytt` schemas are currently in the **experimental** phase. To use schema features, include `--enable-experiment-schema`.
+
+Configuration authors use Schema to verify data value YAML structures and their values and define default data value configuration.
```

Comment:
Mentioning persona is great. 👍🏼 

Thinking pedantically about what action the Configuration Author takes... they are responsible for _declaring_ the ~~variables~~ data values.

```suggestion
Configuration Authors use Schema to declare data values; specifying the type and default value for each.
```